### PR TITLE
Fix text for loc-rib to reflect that it's only for one route.

### LINKED ISSUE #337: Fix loc-rib text in Internet-Draft
Noted by Reshad Rahman on idr@ietf.org:

> Section [2.3.1](https://datatracker.ietf.org/doc/html/draft-ietf-idr-bgp-model/#section-2.3.1) states "The loc-rib table may contain multiple routes for a given prefix, with an attribute to indicate which was selected as the best-path." However, although the best-path indicator exists in adj-rib-in-post (via grouping bgp-adj-rib-in-route-annotations-state), it is not present in loc-rib. Oversight?

His observation is correct.  This is a leftover from the OC model and doesn't match how BGP works.

### PR DESCRIPTION
**Pull request by jhaas-pfrc**
Closes #337

### CODE REVIEW

File: draft/draft-ietf-idr-bgp-model.xml [hunk 1]
```diff
@@ -655,10 +658,10 @@ reference: RFC XXXX
           context.</li>
 
           <li>ietf-bgp-rib - grouping for representing BGP RIBs.</li>
-	  <li>ietf-bgp-rib-attributes - common data definitions for BGP
-	  attributes used in BGP RIB tables.</li>
-	  <li>ietf-bgp-rib-tables - structural data definitions for BGP
-	  routing tables.</li>
+          <li>ietf-bgp-rib-attributes - common data definitions for BGP
```
Comment (mjethanandani) [inline draft/draft-ietf-idr-bgp-model.xml:661]: I do not understand these changes. Are these formatting changes?

File: draft/draft-ietf-idr-bgp-model.xml [hunk 2]
```diff
@@ -711,78 +714,78 @@ reference: RFC XXXX
       </ul></t>
 
       <section title="Modules and submodules for base items">
-	<section title="ietf-bgp module">
```
Comment (mjethanandani) [inline draft/draft-ietf-idr-bgp-model.xml:714]: Same for this and all the subsequent changes for each module name.

File: draft/draft-ietf-idr-bgp-model.xml [hunk 3]
```diff
@@ -1077,31 +1080,31 @@ INSERT_TEXT_FROM_FILE(../src/yang/example-bgp-configuration-a.1.3.xml,68)
...
       <section title="BGP Policy - Match Prefix and Set Community">
         <t>Routing policy using community value involves configuring
-	rules to match community values in the inbound or outbound
-	direction. In this example, which is heavily borrowed from
-	the example on the Cisco community page, we look at
-	"match community exact" match, which happens only when BGP
-	updates have the same community values as specified in the
-	community list.</t>
-	<t>The topology in this example consists of three routers,
+        rules to match community values in the inbound or outbound
```
Comment (mjethanandani) [inline draft/draft-ietf-idr-bgp-model.xml:1083]: And these changes.

File: draft/draft-ietf-idr-bgp-model.xml (hunk 1 above)
Comment (jhaas-pfrc) [reply draft/draft-ietf-idr-bgp-model.xml:661]: Errant tabs to spaces changes throughout the file.  We really need something in the build that catches it.

**Review by mjethanandani — APPROVED** [review]

**Review by keyur-arrcus — COMMENTED** [review]
Looks good.

**Review by suehares — COMMENTED** [review]
These changes look fine.

**Review by suehares — COMMENTED** [review]
This change looks fine. 

Sue

**Pull request merged**